### ISSUE TITLE
[Kura 2.1.0 QA] ExamplePublisher: Snapshots are not created

### ISSUE THREAD

**Issue by djurod** [eclipse/kura#935]
Snapshots are not created when either publish.rate or publish.retain is changed (I didn't test with other fields).

Kura 2.1.0 build 15 on BeagleBone Black

**Comment by cdealti** [comment]
@djurod how did you check the snapshot was not created? From the Web UI, looking at the /opt/eclipse/kura/data/snapshots, other?

**Comment by djurod** [comment]
Using the Web UI

**Comment by MMaiero** [comment]
Could you send us the kura.log and kura-console.log files?

**Comment by djurod** [comment]
It appears this is works in build 16.

Should we close this issue or should I reload build 15 on BBB and try to reproduce the problem?

**Comment by MMaiero** [comment]
Could you try to reload build 15 and, if that happens again, send the log files? Thanks

**Comment by djurod** [comment]
I loaded build 15 on it again and I wasn't able to reproduce the problem.

I did notice something weird in the Web UI. The date and time of the last new snapshots (in "Created On"  column) was set exactly the same value. Through the log I noticed that this is probably done by design:

```
2016-05-21 22:38:28,170 [qtp2061641-32 - /denali/component] WARN  o.e.k.c.c.ConfigurationServiceImpl - Snapshot ID: 1463870308169 is in the past. Adjusting ID to: 1481905012480 + 1
```

The reason for this is that I didn't set a working NTP server in kura after installation, so the device wasn't able to get the correct date after it was restarted.

From what I see my BBB was probably in some weird state when I was doing these tests the first time. As far as I am concerned this issue can be closed.
